lib/ansible/utils/display.py

```diff
@@ -130,9 +113,9 @@ def get_text_width(text):
             w = 0
         width += w
 
-    if width == 0 and counter and not _LOCALE_INITIALIZED:
+    if width == 0 and counter and not LOCALE_INITIALIZED:
         raise EnvironmentError(
-            'ansible.utils.display.initialize_locale has not been called, '
+            'ansible.utils.locale.initialize_locale has not been called, '
             'and get_text_width could not calculate text width of %r' % text
         )
```

Comment:
Can we remove these checks now that failure to initialize the locale is an error at startup?

Reply:
It seems strange to me that we'd keep the exception (and the exception handler) for an unsupported use case. However, since it's staying, it would be good to have unit test coverage for this. Since it was untested before, tests could be added in another PR.

I would at least add a comment that mentions this code path should only be reachable by third-party API usage.

Reply:
Hrm, let me think about it again.  I was thinking I needed to keep the `LOCALE_INITIALIZED` check in the `if` statement, but I didn't think about it too hard.  I think you are right, and I should just remove that check altogether.

Reply:
Removed.